.circleci/scripts/test-run-e2e.sh

```diff
@@ -20,7 +20,7 @@ then
 fi
 
 # Run the actual test command from the parameters
-timeout 20m "$@" --retries 1
+timeout 25m "$@" --retries 1
```

Comment:
I don't intend to check this in. But the feature that runs e2e tests extra times when their file was edited is exceeding the 20 minute timeout.   I suggested increasing it here: https://github.com/MetaMask/metamask-extension/pull/27146